libs/entry-components/theming-configuration.md

```diff
@@ -350,32 +352,93 @@ $custom-theme: (
 
 ### 6. Buttons
 
-Additional customization  for buttons:
+Additional customization for buttons:
 
-- `icon-size` - sets the default size for icons on buttons when icons are present.
-
-Code Example of general submap configuration:
+- `icon-size`:  sets the default size for icons on buttons when icons are present.
+- `radio`:  Contains property for radio buttons
+  - `background`: sets the background color for the inner and outer circles of radio buttons
 
 ```scss
 $custom-theme: (
   general: (
     buttons: (
-      icon-size: 48px
+      icon-size: 48px,
+      radio: (
```

Comment:
This is fine and it works fine, but seems our implementation is wrong when we don't specify this.
As far as I can see, default value is null, but we're unconditionally setting it in entry-components\styles\modules\components\buttons\_radio.scss